build.bat

```diff
@@ -59,7 +66,10 @@ set linker_settings=%libs% %linker_flags%
 del *.pdb > NUL 2> NUL
 del *.ilk > NUL 2> NUL
 
-cl %compiler_settings% "src\main.cpp" /link %linker_settings% -OUT:%exe_name%
+rem cl %compiler_settings% "src\main.cpp" /link %linker_settings% -OUT:%exe_name%
+
+cl %compiler_settings% "src\main.cpp" /link %linker_settings% -OUT:%exe_name% ^
+	&& odin run examples/demo
```

Comment:
Could you remove the `&& odin run examples/demo`? there is the check on line 75 that if we're not building release mode we run the demo. For the CI the demo is run explicitly in it's own step

Reply:
I will do, this was an accidental commit